tests/test_environment.py

```diff
@@ -348,3 +349,12 @@ def test_present_non_empty_namespace_packages_metadata_does_warn():
 
 def test_present_but_empty_namespace_packages_metadata_does_not_warn():
   assert_namespace_packages_warning('pycodestyle', '2.5.0', expected_warning=False)
+
+
+def test_can_add_handles_wheel_with_build_tag():
+  # The actual PEXEnviroment is irrelevant; all we need is an instance
+  interpreter = PythonInterpreter.get()
```

Comment:
You don't need the interpreter, just `PexInfo.default()` will do. That said, if you did realy set up a controlled interpreter you could test `can_add` returns `True` which would be a bit better test (tests both that we don't raise like before, and that we parse out the right tags and match as well). For that, you can use the fixture higher in this file:
https://github.com/pantsbuild/pex/blob/85f156548e3ac0d4d9c03246815a4dd0a67a31ed/tests/test_environment.py#L175-L178

And:
```python
def test_can_add_handles_wheel_with_build_tag(python_35_interpreter):
  pex_environment = PEXEnvironment(
    pex="",
    pex_info=PexInfo.default(python_35_interpreter),
    interpreter=python_35_interpreter
  )
  wheel_with_build_tag = 'llvmlite-0.29.0-1-cp35-cp35m-manylinux1_x86_64.whl'
  assert pex_enviroment.can_add(Distribution(wheel_with_build_tag))
```